eZ/Bundle/PlatformBehatBundle/Context/Object/BasicContentContext.php

```diff
@@ -0,0 +1,209 @@
+<?php
+/**
+ * This file is part of the BehatBundle package.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ * @version //autogentag//
+ */
+namespace EzSystems\PlatformBehatBundle\Context\Object;
```

Comment:
Actually, the PlatformBehatBundle was meant as a bundle that contains _features_ used to run our behat scenarios. Example: the TestSiteaccess command.

Contexts should as I see it go to the Bundle they belong to. In this case, the CoreBundle.